cookbook/docs/index.md

```diff
@@ -0,0 +1,377 @@
+:animate: fade-in-slide-down
+
+`pyflyte run` enables you to execute a specific workflow in your Python script
+using the syntax `pyflyte run <path/to/script.py> <workflow_function_name>`.
+
+Keyword arguments can be supplied to ``pyflyte run`` by passing in options in
+the format ``--kwarg value``, and in the case of ``snake_case_arg`` argument
+names, you can pass in options in the form of ``--snake-case-arg value``.
+
+::::{note}
+If you want to run a workflow with `python example.py`, you would have to write
+a `main` module conditional at the end of the script to actually run the
+workflow:
+
+:::{code-block} python
+if __name__ == "__main__":
+    training_workflow(hyperparameters={"C": 0.1, "max_iter": 3000})
+:::
+
+This becomes even more verbose if you want to pass in arguments:
+
+:::{code-block} python
+if __name__ == "__main__":
+    import json
+    from argparse import ArgumentParser
+
+    parser = ArgumentParser()
+    parser.add_argument("--hyperparameters", type=json.loads)
+    ...  # add the other options
+
+    args = parser.parse_args()
+    training_workflow(hyperparameters=args.hyperparameters)
+:::
+
+::::
+
+:::::
+
+
+## Flyte Cluster Execution
+
+To run your Flyte workflows on an actual Flyte cluster, first we need to
+spin up a local demo cluster:
+
+```{prompt} bash $
+flytectl demo start
+```
+
+````{div} shadow p-3 mb-8 rounded
+**Expected Output:**
+
+```{code-block}
+👨‍💻 Flyte is ready! Flyte UI is available at http://localhost:30080/console 🚀 🚀 🎉
+Add KUBECONFIG and FLYTECTL_CONFIG to your environment variable
+export KUBECONFIG=$KUBECONFIG:/Users/<username>/.kube/config:/Users/<username>/.flyte/k3s/k3s.yaml
+export FLYTECTL_CONFIG=/Users/<username>/.flyte/config-sandbox.yaml
+```
+
+```{important}
+Make sure to export the `KUBECONFIG` and `FLYTECTL_CONFIG` environment variables
+in your shell.
+```
+````
+
+
+### Remote Execution
+
+Run the workflow on the Flyte cluster with `pyflyte run` using the `--remote`
+flag:
+
+```{prompt} bash $
+pyflyte run --remote example.py training_workflow --hyperparameters '{"C": 0.1, "max_iter": 3000}'
+```
+
+````{div} shadow p-3 mb-8 rounded
+
+**Expected Output:** A URL to the workflow execution on your demo Flyte cluster:
+
+```{code-block}
+Go to http://localhost:30080/console/projects/flytesnacks/domains/development/executions/<execution_name> to see execution in the console.
+```
+
+Where ``<execution_name>`` is a unique identifier for the workflow execution.
+
+````
+
+Unlike the previous ``pyflyte run`` invocation, passing the ``--remote`` flag
+will trigger the execution on the configured backend.
+
+
+## Inspect the Results
+
+Navigate to the URL produced as the result of running `pyflyte run`. This will
+take you to FlyteConsole, the web UI used to manage Flyte entities such as tasks,
+workflows, and executions.
+
+![getting started console](https://github.com/flyteorg/static-resources/raw/main/flytesnacks/getting_started/getting_started_console.gif)
```

Comment:
The GIF isn't available.

Reply:
let's address this in a different PR... unfortunate that things aren't loading 😕